app/src/main/java/com/rockthevote/grommet/data/db/relationship/SessionWithRegistrations.kt

```diff
@@ -9,10 +9,10 @@ import com.rockthevote.grommet.data.db.model.Session
  * Created by Mechanical Man on 5/3/20.
  */
 data class SessionWithRegistrations(
-        @Embedded val session: Session,
-        @Relation(
-                parentColumn = "session_id",
-                entityColumn = "session_id"
-        )
-        val registrations: List<Registration>
+        @Embedded val session: Session //, TODO Readd foreign key requirements
+//        @Relation(
```

Comment:
the relationship shouldn't impact the foreign key constraint, it's just a construct that just makes querying data easier.

was it causing problems?

Reply:
I'll double check on this one, I think I removed them both just because it was erroring in the `Registration` implementation, but can't recall if this implementation actually mattered